tests/stub/01-check-stdin.php

```diff
@@ -1,10 +1,12 @@
 <?php
 
+namespace Clue\Tests\React\Stdio\stub;
```

Comment:
Not sure these stub classes should have a namespace. What do you think? :+1:

Reply:
Fixed :+1: